src/Drivers/Pgsql/PgsqlDriver.php

```diff
@@ -238,7 +238,7 @@ public function convertToSql($value, $type)
 						$value->setTimezone($this->simpleStorageTz);
 					}
 				}
-				return "'" . $value->format('Y-m-d H:i:s') . "'";
+				return "'" . $value->format('Y-m-d H:i:s') . "'::timestamptz";
```

Comment:
That was original implementation. I choose to rebase it with this simply for it's semantic value: I interpret *tz* as *the timezone is known* (in contrast to timestamp, which might indicate *a time from application in unknown tz*).

Without the timezone specified in the format string, those are all equivalent (as in *interchangeable*)
```sql
select '2015-01-01 16:00:00'::timestamp;
select '2015-01-01 16:00:00'::timestamptz::timestamp;
select '2015-01-01 16:00:00'::timestamptz;
select '2015-01-01 16:00:00'::timestamp::timestamptz;
```

but I wouldn't mind `timestamp` either; is there a reason you requested `timestamp`?